files/es/learn/common_questions/web_mechanics/what_are_hyperlinks/index.md

```diff
@@ -12,12 +12,12 @@ En este artículo, repasaremos qué son los hipervínculos y por qué son import
       <th scope="row">Prerrequisitos:</th>
       <td>
         Debes saber
-        <a href="/en-US/Learn/How_the_Internet_works">how the Internet works</a>
+        <a href="/es/Learn/How_the_Internet_works">como funciona internet</a>.
```

Comment:
Should we add the question marks?

Reply:
Honestly, I'm not pretty sure, there's no marks in `en-US`

Reply:
I know, the way it is structured in English makes me think it should has question marks but if you change it as you said will be better

Reply:
In Spanish the difference is in the accent in the word "como" or "cómo" and the obvious question marks, also how is it structured in English suggest that it is not a question, in that case it should says some like "how does the internet work?"